go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -172,6 +215,13 @@ func Init() (err error) {
 	}
 	log.Info("SBI Initialized")
 
+	distancePeriodicTicker = time.NewTicker(time.Second)
```

Comment:
Move this code inside the Run() function.
Also, stop the ticker in the Stop() function.

Reply:
moved